cpp/tests/community/louvain_test.cpp

```diff
@@ -383,6 +383,9 @@ TEST_P(Tests_Louvain_File64, CheckInt64Int64FloatFloat)
     override_File_Usecase_with_cmd_line_arguments(GetParam()));
 }
 
+#if 0
+// FIXME:  We should use these tests, gtest-1.11.0 makes it a runtime error
+//         to define and not instantiate these.
```

Comment:
Shouldn't we better run this? Or is this test failing?

Reply:
There is no current RMAT test defined below.  We create the definition here, but we aren't actually using it below.

gtest 1.10 (which we are currently using) ignored this.
gtest 1.11 (which is used by DLFW and I assume we will migrate to shortly) treats the definition of a `TEST_P` and not using it as an error.

This is the expedient solution (ifdef out the offending definitions that aren't used).  I added the FIXME to identify that we should actually be using these tests.  But I needed a simple fix to unblock DLFW and didn't want to spend the time to add and debug the Rmat tests for these algorithms.